### ISSUE TITLE
Not enough available nodes with replicas-on-different set

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-server#95]
Hi, we have a problem with replicas-on-different flag, linstor just stopped placing new resources with this flag set:
```bash
# linstor rd c test
SUCCESS:
Description:
    New resource definition 'test' created.
Details:
    Resource definition 'test' UUID is: b630b03a-f77c-40d6-a283-ba8fb6a51875
```

```bash
# linstor vd c test 16G
SUCCESS:
    New volume definition with number '0' of resource definition 'test' created.
```

```bash
# linstor resource create --replicas-on-same opennebula-1 --replicas-on-different moonshot --auto-place 2 --storage-pool thindata test              
ERROR:
Description:
    Not enough available nodes
Details:
    Not enough nodes fulfilling the following auto-place criteria:
     * has a deployed storage pool named 'thindata'
     * the storage pool 'thindata' has to have at least '16777216' free space
     * the current access context has enough privileges to use the node and the storage pool
     * the node is online
    Auto-placing resource: test
Show reports:
    linstor error-reports show 5DEE75F9-00000-000027
```

[5DEE75F9-00000-000027.log](https://github.com/LINBIT/linstor-server/files/3943986/5DEE75F9-00000-000027.log)

Then I've checked it manually using this oneliner:
```bash
# Select online nodes with aux-properties `opennebula-1` and `moonshot` set.
# For each node query storagepool `thindata`.
# Print aux-properties and storagepool line from the output
linstor -m n l | jq -rc '.[].nodes[] | select(.connection_status==2) | select(.props | from_entries | select(."Aux/opennebula-1") | select(."Aux/moonshot")) | "\(.name) \(.props | from_entries | "\(."Aux/opennebula-1") \(."Aux/moonshot")")"' | while read NODE AUX1 AUX2; do echo -n "opennebula-1=$AUX1 moonshot=$AUX2 "; linstor sp l -n $NODE -s thindata | grep thindata; done
```
In the output I see a lot of free nodes which satisfies requested options `--replicas-on-same opennebula-1 --replicas-on-different moonshot --auto-place 2` and having requested `16G` drive space:
```bash
opennebula-1=true moonshot=10 | thindata    | m10c10 | LVM_THIN | data/thindata |   159.84 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=10 | thindata    | m10c11 | LVM_THIN | data/thindata |   474.39 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=10 | thindata    | m10c12 | LVM_THIN | data/thindata |   419.94 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=10 | thindata    | m10c13 | LVM_THIN | data/thindata |   494.91 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=10 | thindata    | m10c14 | LVM_THIN | data/thindata |   419.94 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=10 | thindata    | m10c15 | LVM_THIN | data/thindata |   419.94 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=10 | thindata    | m10c16 | LVM_THIN | data/thindata |   419.94 GiB |       900 GiB | True              | Ok    |
...truncated...
opennebula-1=true moonshot=15 | thindata    | m15c43 | LVM_THIN | data/thindata |   512.91 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=15 | thindata    | m15c44 | LVM_THIN | data/thindata |   492.93 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=15 | thindata    | m15c45 | LVM_THIN | data/thindata |   532.71 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=15 | thindata    | m15c5 | LVM_THIN | data/thindata |   514.89 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=15 | thindata    | m15c6 | LVM_THIN | data/thindata |   466.92 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=15 | thindata    | m15c7 | LVM_THIN | data/thindata |   524.88 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=15 | thindata    | m15c8 | LVM_THIN | data/thindata |   534.87 GiB |       900 GiB | True              | Ok    |
opennebula-1=true moonshot=15 | thindata    | m15c9 | LVM_THIN | data/thindata |   544.95 GiB |       900 GiB | True              | Ok    |
```

What's wrong with that?

**Comment by kvaps** [comment]
I can append that manually creation is working fine:
```bash
# linstor r c m10c13 m15c6 test --storage-pool thindata     
SUCCESS:
Description:
    New resource 'test' on node 'm10c13' registered.
Details:
    Resource 'test' on node 'm10c13' UUID is: 7f62c03d-3378-459d-ad9e-f468fca275b3
SUCCESS:
Description:
    Volume with number '0' on resource 'test' on node 'm10c13' successfully registered
Details:
    Volume UUID is: 0779f1ff-d0fb-4f7b-92f5-dc4d441c84fe
SUCCESS:
Description:
    New resource 'test' on node 'm15c6' registered.
Details:
    Resource 'test' on node 'm15c6' UUID is: 79210e85-4cd2-4c9e-8c16-a8e2cb743806
SUCCESS:
Description:
    Volume with number '0' on resource 'test' on node 'm15c6' successfully registered
Details:
    Volume UUID is: 9da08f5d-49e1-43fa-9e1d-d299f2ec98f8
SUCCESS:
    Created resource 'test' on 'm15c6'
SUCCESS:
    Created resource 'test' on 'm10c13'
SUCCESS:
Description:
    Resource 'test' on 'm15c6' ready
Details:
    Node(s): 'm10c13', 'm15c6', Resource: 'test'
SUCCESS:
Description:
    Resource 'test' on 'm10c13' ready
Details:
    Node(s): 'm10c13', 'm15c6', Resource: 'test'
```

**Comment by kvaps** [comment]
Hi, I just wanted to mention that issue is still there and not yet fixed in **1.3.0**:

 Another report:

```bash
# linstor rd c testoz
SUCCESS:
Description:
    New resource definition 'testoz' created.
Details:
    Resource definition 'testoz' UUID is: 0bd254ed-0605-4d33-8d8c-fe45837af9d7

# linstor vd c testoz 10G
SUCCESS:
    New volume definition with number '0' of resource definition 'testoz' created.

# linstor resource create --replicas-on-same opennebula-1 --replicas-on-different moonshot --auto-place 2 --storage-pool thindata testoz            
ERROR:
Description:
    Not enough available nodes
Details:
    Not enough nodes fulfilling the following auto-place criteria:
     * has a deployed storage pool named 'thindata'
     * the storage pool 'thindata' has to have at least '10485760' free space
     * the current access context has enough privileges to use the node and the storage pool
     * the node is online
    Auto-placing resource: testoz
Show reports:
    linstor error-reports show 5DF8597F-00000-000195
```

[5DF8597F-00000-000195.log](https://github.com/LINBIT/linstor-server/files/4025966/5DF8597F-00000-000195.log)

For check this problem I made a small bash script, which is checking all the nodes which met these auto-place criteria.

```bash
while read node size; do
  echo -n "$node $size "
  linstor -m n l -n "$node" | jq -rc '.[].nodes[] | [ (.props[] | select(.key | contains("Aux"))) ] | from_entries' 
done < <(linstor -m sp l | jq -r '.[].stor_pools[] | select(.stor_pool_name=="thindata") | select(.free_space.free_capacity>104857600) | "\(.node_name) \(.free_space.free_capacity)"')
```
This script is selecting all the storage pools with `thindata` name and more than `104857600` free_capacity and queries the aux-properties for their nodes.

<details>
<summary>The output of this script</summary>
<pre>
# NAME SIZE AUX-PROPERTIES
m10c1 914557501 {"Aux/moonshot":"10"}
m10c10 167604387 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c11 445151969 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c12 440339005 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c13 518950748 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c14 440339005 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c15 440339005 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c16 440339005 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c17 573686415 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c18 579537469 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c19 571515863 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c2 545563607 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c20 583029227 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c21 584350433 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c22 587275960 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c23 585482895 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c24 577461288 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c25 566136668 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c26 459119001 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c27 548300390 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c28 566702899 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c29 566608527 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c3 564909834 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c30 574535761 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c31 554812047 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c32 577555660 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c33 569250938 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c34 578688122 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c35 586143498 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c36 528387932 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c37 336152494 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c38 568590336 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c39 579726213 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c4 574063902 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c40 565192949 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c41 472331059 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c42 550376570 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c43 573875159 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c44 527349841 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c45 574252646 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c5 584067317 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c6 582085509 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c7 587087216 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c8 424578908 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m10c9 540278784 {"Aux/moonshot":"10","Aux/opennebula-1":"true"}
m13c10 246216130 {"Aux/moonshot":"13"}
m13c11 168359362 {"Aux/moonshot":"13"}
m13c12 201200762 {"Aux/moonshot":"13"}
m13c13 146559467 {"Aux/moonshot":"13"}
m13c14 189121167 {"Aux/moonshot":"13"}
m13c15 165716951 {"Aux/moonshot":"13"}
m13c16 126647009 {"Aux/moonshot":"13"}
m13c17 189121167 {"Aux/moonshot":"13"}
m13c18 199030210 {"Aux/moonshot":"13"}
m13c2 222340055 {"Aux/moonshot":"13"}
m13c21 174682275 {"Aux/moonshot":"13"}
m13c22 294628884 {"Aux/moonshot":"13"}
m13c24 151938662 {"Aux/moonshot":"13"}
m13c25 151844290 {"Aux/moonshot":"13"}
m13c26 129855651 {"Aux/moonshot":"13"}
m13c27 142595850 {"Aux/moonshot":"13"}
m13c28 242063769 {"Aux/moonshot":"13"}
m13c29 180155842 {"Aux/moonshot":"13"}
m13c30 162319564 {"Aux/moonshot":"13"}
m13c31 176192225 {"Aux/moonshot":"13"}
m13c33 142784593 {"Aux/moonshot":"13"}
m13c34 302933606 {"Aux/moonshot":"13"}
m13c35 136744796 {"Aux/moonshot":"13"}
m13c36 137216655 {"Aux/moonshot":"13"}
m13c37 135329218 {"Aux/moonshot":"13"}
m13c38 153731727 {"Aux/moonshot":"13"}
m13c39 176569712 {"Aux/moonshot":"13"}
m13c4 179117752 {"Aux/moonshot":"13"}
m13c40 134291128 {"Aux/moonshot":"13"}
m13c41 165622579 {"Aux/moonshot":"13"}
m13c42 163546398 {"Aux/moonshot":"13"}
m13c43 382017208 {"Aux/moonshot":"13"}
m13c44 255936430 {"Aux/moonshot":"13"}
m13c5 251500953 {"Aux/moonshot":"13"}
m13c6 138915348 {"Aux/moonshot":"13"}
m13c8 139104092 {"Aux/moonshot":"13"}
m13c9 230550405 {"Aux/moonshot":"13"}
m14c10 204315033 {"Aux/moonshot":"14"}
m14c12 172794839 {"Aux/moonshot":"14"}
m14c13 235457740 {"Aux/moonshot":"14"}
m14c14 139198464 {"Aux/moonshot":"14"}
m14c15 162319564 {"Aux/moonshot":"14"}
m14c16 141935247 {"Aux/moonshot":"14"}
m14c17 139104092 {"Aux/moonshot":"14"}
m14c19 158261575 {"Aux/moonshot":"14"}
m14c2 160243384 {"Aux/moonshot":"14"}
m14c20 219225784 {"Aux/moonshot":"14"}
m14c21 213752217 {"Aux/moonshot":"14"}
m14c22 124382085 {"Aux/moonshot":"14"}
m14c23 136178565 {"Aux/moonshot":"14"}
m14c24 163357655 {"Aux/moonshot":"14"}
m14c25 147503185 {"Aux/moonshot":"14"}
m14c26 134291128 {"Aux/moonshot":"14"}
m14c27 246404874 {"Aux/moonshot":"14"}
m14c28 147691929 {"Aux/moonshot":"14"}
m14c3 166094438 {"Aux/moonshot":"14"}
m14c30 151278059 {"Aux/moonshot":"14"}
m14c31 223850004 {"Aux/moonshot":"14"}
m14c33 181477048 {"Aux/moonshot":"14"}
m14c35 200068300 {"Aux/moonshot":"14"}
m14c36 157600972 {"Aux/moonshot":"14"}
m14c37 191763578 {"Aux/moonshot":"14"}
m14c39 137216655 {"Aux/moonshot":"14"}
m14c4 291325870 {"Aux/moonshot":"14"}
m14c40 154958561 {"Aux/moonshot":"14"}
m14c41 166283182 {"Aux/moonshot":"14"}
m14c42 271979642 {"Aux/moonshot":"14"}
m14c44 147691929 {"Aux/moonshot":"14"}
m14c5 129572536 {"Aux/moonshot":"14"}
m14c6 183081369 {"Aux/moonshot":"14"}
m14c7 165433835 {"Aux/moonshot":"14"}
m14c9 158922178 {"Aux/moonshot":"14"}
m15c1 910782627 {"Aux/moonshot":"15"}
m15c10 526311751 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c11 445151969 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c12 555661393 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c13 445246341 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c14 551037173 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c15 541222502 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c16 424578908 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c17 547545415 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c18 580858675 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c19 554812047 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c2 907196497 {"Aux/moonshot":"15"}
m15c20 440339005 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c21 555661393 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c22 441565839 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c23 458080911 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c24 515836477 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c25 455532871 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c26 469688647 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c27 535748935 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c28 466480005 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c29 298686873 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c3 510834769 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c30 539901296 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c31 561606819 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c32 547639787 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c33 517157683 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c34 551509032 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c35 545657978 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c36 575573852 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c37 560851845 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c38 424578908 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c39 529426022 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c4 583972945 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c40 554812047 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c41 560191242 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c42 527349841 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c43 543109939 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c44 516874567 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c45 558586920 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c5 539901296 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c6 513665925 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c7 550376570 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c8 560851845 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
m15c9 571421491 {"Aux/moonshot":"15","Aux/opennebula-1":"true"}
</pre>
</details>

As you can see there is a lot of nodes, which can handle the resource.

**Comment by kvaps** [comment]
Yes, look into the error report, there is some exception maybe is it related with this problem?
```
Category:                           RuntimeException
Class name:                         OnAssemblyException
Class canonical name:               reactor.core.publisher.FluxOnAssembly.OnAssemblyException
Generated at:                       <UNKNOWN>

Error message:                      Assembly site of producer [reactor.core.publisher.MonoFlatMapMany] is identified by light checkpoint [Auto-place resource including thin pools].Error has been observed by the following operator(s):
        |_      Assembly site of producer [reactor.core.publisher.MonoFlatMapMany] is identified by light checkpoint [Auto-place resource].


Error context:
    Not enough available nodes

Call backtrace:

    Method                                   Native Class:Line number
```

**Comment by kvaps** [comment]
Hi, I just wanted to mention that issue is still there and not yet fixed in **1.4.0**:

 Another report:

```bash
# linstor rd c testa
SUCCESS:
Description:
    New resource definition 'testa' created.
Details:
    Resource definition 'testa' UUID is: ee912ff7-281a-487f-9354-b579e10753b2

# linstor vd c testa 10g
SUCCESS:
    New volume definition with number '0' of resource definition 'testa' created.

# linstor r c --auto-place 2 --replicas-on-same=hosting-2-storage --replicas-on-different=moonshot testa -s thindata
ERROR:
Description:
    Not enough available nodes
Details:
    Not enough nodes fulfilling the following auto-place criteria:
     * has a deployed storage pool named 'thindata'
     * the storage pool 'thindata' has to have at least '10485760' free space
     * the current access context has enough privileges to use the node and the storage pool
     * the node is online
    Auto-placing resource: testa
Show reports:
    linstor error-reports show 5E283162-00000-000037
```

[5E283162-00000-000037.log](https://github.com/LINBIT/linstor-server/files/4097383/5E283162-00000-000037.log)


<details>
<summary>There is still a lot of nodes, which can handle the resource.</summary>
<pre>
m16c1 3913542008 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c10 738649964 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c11 3303198425 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c12 3916305530 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c13 3909594120 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c14 3770233651 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c15 3846033113 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c2 3913542008 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c3 3866562134 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c4 3866562134 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c5 3223056285 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c6 3894986932 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c7 3764311818 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c8 3764311818 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m16c9 3570470486 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"16"}
m5c1 946864128 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"5"}
m5c2 3570470486 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"5"}
m6c1 738255175 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c10 3303198425 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c11 3916305530 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c12 3909594120 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c13 3770233651 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c14 3846033113 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c15 3787999150 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c2 3787999150 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c3 3896171298 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c4 3896171298 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c5 3775365906 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c6 3775365906 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c7 3876431855 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c8 3873668333 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m6c9 3886696366 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"6"}
m9c1 3886696366 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c10 3699566444 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c11 3699566444 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c12 3816423948 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c13 3816423948 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c14 3588235984 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c15 3588235984 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c2 3791157460 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c3 3791157460 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c4 410185629 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c5 454401982 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c6 750493630 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c7 750098841 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c8 3374655209 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
m9c9 3374655209 {"Aux/hosting-2":"true","Aux/hosting-2-storage":"true","Aux/moonshot":"9"}
</pre>
</details>

**Comment by kvaps** [comment]
Not sure, but it seems issue is no longer persists on 1.6.2, I'll reopen it if  we will face with this problem again
